proto/validator/accounts/v2/web_api.proto

```diff
@@ -226,19 +228,19 @@ message CreateAccountRequest {
     uint64 num_accounts = 1;
 }
 
+message DepositMessage {
```

Comment:
Please add this to ssz static spec tests: https://github.com/prysmaticlabs/prysm/blob/63149a3dc3787e73380d8f807767d2b5124960c0/proto/testing/ssz_static_test.go#L109-L111